languagedetector/language-detector/src/main/webapp/WEB-INF/liferay-display.xml

```diff
@@ -2,8 +2,7 @@
 <!DOCTYPE display PUBLIC "-//Liferay//DTD Display 6.2.0//EN" "http://www.liferay.com/dtd/liferay-display_6_2_0.dtd">
 
 <display>
-	
-	<category name="category.sample">
+	<category name="hidden">
 		<portlet id="language-detector"></portlet>
```

Comment:
Can this portlet ever be added to a page? If not, make it a system portlet. See liferay-portlet-app_6_2_0.dtd